dev/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM quay.io/cloudnativetoolkit/mkdocs
```

Comment:
out of interest, why cant we FROM the regular material image here?

Reply:
yes we can use the material one

Reply:
yes we can use the material one